packages/chevrotain/src/parse/parser/parser.ts

```diff
@@ -72,7 +72,7 @@ export const DEFAULT_PARSER_CONFIG: IParserConfig = Object.freeze({
     outputCst: true,
     errorMessageProvider: defaultParserErrorProvider,
     serializedGrammar: null,
-    nodePositionTracking: <"full" | "onlyStart" | "onlyOffset" | "none">"none"
+    nodePositionTracking: <IParserConfig["nodePositionTracking"]>"none"
```

Comment:
This is better but I meant something like this:
https://github.com/SAP/chevrotain/blob/master/packages/chevrotain/src/api.ts#L144

You can reference a type from the api.d.ts (type of x.y.z)

Reply:
I got this error, as IParserConfig is an interface, and not a type (I guess, I'm not a TypeScript expert)

![Capture d’écran de 2019-05-27 10-31-12](https://user-images.githubusercontent.com/25530077/58406780-a8e55000-806a-11e9-826a-4620573f554b.png)

Reply:
you can define a custom type in api.d.ts

```typescript
export type NodePositionTrackingOptioins = "A" | "B" | "C" and reuse that type 
else where in the d.ts file or the .ts files
```